relay-server/src/actors/global_config.rs

```diff
@@ -164,6 +170,8 @@ impl GlobalConfigService {
             internal_rx,
             upstream,
             fetch_handle: SleepHandle::idle(),
+            last_fetched: Instant::now(),
+            upstream_failure_interval: Duration::from_secs(35),
```

Comment:
35 is arbitrary, but should have time for 3 requests (after 10s each).

Reply:
Should this be in `Config`?

Reply:
We may not need it for now, but I'll keep this in mind to address it in the future.